target/scripts/update-check.sh

```diff
@@ -0,0 +1,55 @@
+#! /bin/bash
+
+# DMS_VERSION == Commit hash?
+if [[ ${DMS_VERSION} =~ ^[a-f0-9]{40}$ ]]
+then
+  DATE=$(date '+%F %T')
+  echo "${DATE} Info: You are using an edge build. Update checks are not supported."
+  exit 0
+fi
+
+API="https://api.github.com/repos/docker-mailserver/docker-mailserver/releases/latest"
```

Comment:
Does latest release always show the "most recent" release or the "greatest" one? So we had a moment where we had to rerun a job on a tag as 9.2 got published by accident and 9.1 was the lastest/most recent and 9.2 was the latest/greatest in this moment

Reply:
Good question, idk.

Reply:
I've tested it..

> Does latest release always show the "most recent" 

yes

I also encounter another problem using the API: There are rate limits in place. In case you already use the API for other things from the same host/IP as DMS runs on, you can hit this limit "easily" (at least in my case). This cost me some time yesterday figuring out, why the update check suddenly stopped working on my machine 😞 There are however no limits for requesting "raw" files from a repo.

I don't think I have mentioned it before^^ --> But I prefer a FILE based solution. The only valid concern IMO was, that we have to remember, to bump that FILE on a new release. I don't think that's a big deal, as we have the same situation with the CHANGELOG file. We also have to remember to adjust it in the last commit before release. One more file wouldn't hurt in my opinion. It's simple, it's flat and it just works. No rate limits and a lot of [big projects](https://github.com/home-assistant/version/blob/master/stable.json) do it successfully in a similar way.

Both mentioned problems would not relate to a file based solution 😉

PS: That was my last try to convince you all 😄 If you are unsure, take a look at my summary post [here](https://github.com/docker-mailserver/docker-mailserver/issues/1946#issuecomment-836905633), which covers all mentioned concerns from you.

Reply:
I think that since you're the creator of the PR and since you have taken your time to implement it, I will not reject a file-based solution if the benefits outweigh the disadvantages - which they seem to do.

Feel free to "revert" to the file-based approach, I consent :)

Reply:
> The only valid concern IMO was, that we have to remember, to bump that FILE on a new release.

If we go the way I mentioned yesterday with the feature-freeze PR in front of every release we can cover the Update in that PR and multiple people can review if all final version changes (CHANGELOG, VERSION,..) are made. 👍🏻

Reply:
Let's call this PR the last one before feature-freeze and we will directly see how it works out :)